site/content/how-to/control-plane-configuration.md

```diff
@@ -22,42 +22,22 @@ If the resource is invalid to the OpenAPI schema, the Kubernetes API server will
 
 Additionally, the control plane updates the status of the resource (if it exists) to reflect whether it is valid or not.
 
-### Spec
-
-{{< bootstrap-table "table table-striped table-bordered" >}}
-| name    | description                                                     | type                     | required |
-|---------|-----------------------------------------------------------------|--------------------------|----------|
-| logging | Logging defines logging related settings for the control plane. | [logging](#speclogging) | no       |
-{{< /bootstrap-table >}}
-
-### Spec.Logging
-
-{{< bootstrap-table "table table-striped table-bordered" >}}
-| name  | description                                                            | type   | required |
-|-------|------------------------------------------------------------------------|--------|----------|
-| level | Level defines the logging level. Supported values: info, debug, error. | string | no       |
-{{< /bootstrap-table >}}
+**For a full list of configuration options that can be set, see the `NginxGateway spec` in the [API reference]({{< relref "reference/api.md" >}}).**
 
 ## Viewing and Updating the Configuration
 
-{{< note >}} For the following examples, the name `nginx-gateway-config` should be updated to the name of the resource created for your installation. {{< /note >}}
+{{< note >}} For the following examples, the name `ngf-config` should be updated to the name of the resource created for your installation.{{< /note >}}
 
-To view the current configuration:
+To view the current configuration and its status:
 
 ```shell
-kubectl -n nginx-gateway get nginxgateways nginx-gateway-config -o yaml
+kubectl -n nginx-gateway describe nginxgateways ngf-config
 ```
```

Comment:
Do you think it would be helpful to give an example of the output here?

Reply:
So no status output here?

Reply:
Oh whoops, missed that one.